vtds_cluster_kvm/private/__init__.py

```diff
@@ -30,11 +30,31 @@
 )
 CONFIG_DIR = path_join(dirname(__file__), 'config')
 DEPLOY_SCRIPT_NAME = 'deploy_cluster_to_blade.py'
```

Comment:
REVIEWERS: in order to refactor the deployment script nicely, I broke it into multiple files. This code sets up a list of python library files that need to be copied onto the Virtual Blade(s) in the vTDS system to allow deployment to work. These all need to land in the same directory as the main deployment script. Currently I am piling all of this stuff into `/root` to keep things simple.